packages/ckeditor5-page-break/tests/pagebreakediting.js

```diff
@@ -252,7 +252,7 @@ describe( 'PageBreakEditing', () => {
 			it( 'should convert', () => {
 				setModelData( model, '<pageBreak></pageBreak>' );
 
-				expect( getViewData( view, { withoutSelection: true } ) ).to.equal(
+				expect( getViewData( view, { withoutSelection: true, renderUIElements: true } ) ).to.equal(
```

Comment:
I would add this as a separate check (could be inside the same test) so one with `renderUIElements` and another one without it.